compiler/src/dotty/tools/dotc/cc/Capability.scala

```diff
@@ -109,11 +119,11 @@ object Capabilities:
    *
    *  Reach capabilities cannot wrap read-only capabilities or maybe capabilities.
    *  We have
-   *      (x.rd).reach = x*.rd
-   *      (x.rd)?      = (x*)?
+   *      (x?).reach        = (x.reach)?
+   *      (x.rd).reach      = (x.reach).rd
+   *      (x.only[T]).reach = (x*).only[T]
```

Comment:
```suggestion
   *      (x.only[T]).reach = (x.reach).only[T]
```
or use `.*` consistently

Reply:
There's a reason for the difference: `.reach` is the operation that converts to `*` somewhere inside the capability.